.github/workflows/test.yml

```diff
@@ -29,14 +27,23 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: true
-    - uses: 0ad-matters/gh-action-build-pyromod@trunk
+    - uses: 0ad-matters/gh-action-build-pyromod@delay
```

Comment:
Trying to find out how to set the correct branch name for this...

https://github.com/orgs/community/discussions/35812